zkvm/entrypoint/src/lib.rs

```diff
@@ -32,22 +37,81 @@ macro_rules! entrypoint {
 #[cfg(all(target_os = "zkvm", feature = "libm"))]
 mod libm;
 
+const PI_DIGEST_NUM_WORDS: usize = 8;
+
+#[derive(Serialize, Deserialize)]
+pub struct PiDigest<T>(pub [T; PI_DIGEST_NUM_WORDS]);
```

Comment:
seems unnecessary to make this generic in T

Reply:
The digest will be represented either as `u32s` (in the guest program and the runtime) or Word<F> (in the air and prover/verifier).  

So the generic could be useful?

Reply:
Ah I see, it's a bit weird to import `sp1-zkvm-entrypoint` into Runtime though.

Reply:
Ya, agreed.  I'll see if there is a shared library between `core` and `zkvm` (or maybe it makes sense for `zkvm` to import `core`).